### PR TITLE
Create kamma-in-buddhism_buddhadasa.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tags:
+  - karma
+  - rebirth
+year: 2017
+publisher: Liberation Park
+address: "Wisconsin"
+volume: 
+series: Messages from Suan Mokkh Series
+number: 4
+pages: 22
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:20]: only 8 pages of actual content... And even those are half size (A5) so I'd honestly say `pages: 4`

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+formats: [pdf]
+course: view
+tags:
+  - karma
+  - rebirth
+year: 2017
+publisher: Liberation Park
+address: "Wisconsin"
+volume: 
+series: Messages from Suan Mokkh Series
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:18]: in quotes, and "Series" is redundant

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+external_url: "https://www.suanmokkh.org/system/books/files/000/000/089/original/Buddhadas-Kamma-in-buddhism.pdf?1599464247"
+drive_links:
+  - "https://drive.google.com/file/d/1OdOqbmOuzO48q0a7tR508jvGMIgo2REX/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - karma
+  - rebirth
+year: 2017
+publisher: Liberation Park
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:15]: Strings in quotes

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+address: "Wisconsin"
+volume:
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:17]: Remove empty fields

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Kamma in Buddhism"
+authors:
+  - buddhadasa
+translator: Santikaro
+external_url: "https://www.suanmokkh.org/system/books/files/000/000/089/original/Buddhadas-Kamma-in-buddhism.pdf?1599464247"
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:6]: please remove the unnecessary `?1599464247` off the end

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/1OdOqbmOuzO48q0a7tR508jvGMIgo2REX/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - karma
+  - rebirth
+year: 2017
+publisher: Liberation Park
+address: "Wisconsin"
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:16]: "Norwalk, WI"?

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  - buddhadasa
+translator: Santikaro
+external_url: "https://www.suanmokkh.org/system/books/files/000/000/089/original/Buddhadas-Kamma-in-buddhism.pdf?1599464247"
+drive_links:
+  - "https://drive.google.com/file/d/1OdOqbmOuzO48q0a7tR508jvGMIgo2REX/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - karma
+  - rebirth
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:13]: Let's not tag rebirth...

File: _content/monographs/kamma-in-buddhism_buddhadasa.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+> We should see the
+truth that the mind that performs a deed is kamma itself and the
+subsequent mind is the result (vipāka) of that kamma. Other
+results that follow it are only uncertain by-products, since they
+may or may not occur, or do not keep up with our expectations
+due to other interfering factors.
+
+In this monograph, the author explains kamma (karma) as it is understood in Buddhism and how to use kamma (action) in a way that leads to the end of action and results in nibanna.
```
Comment (buddhist-uni) [inline _content/monographs/kamma-in-buddhism_buddhadasa.md:30]: Idk man... it's so short I'm inclined to move this to essays... what do you think?

**Review by buddhist-uni — COMMENTED** [review]
Sādhu 😁

**Comment by JJM317** [comment]
Ya, I was going to put this in essays, actually, but I saw it had a
publisher (instead of self-published), so going by the guide I stuck in
monographs. Happy to move it to essays. Let me know.

--------------------

**Comment by buddhist-uni** [comment]
Yeah, let's go ahead and put it in essays despite the publisher.

**Review by buddhist-uni — APPROVED** [review]
Okay, I just went ahead and fixed a couple nits. This is now good to go.

**Pull request merged**